packages/layout/src/container.tsx

```diff
@@ -36,64 +33,40 @@ export const Container = forwardRef<ContainerProps, "div">(function Container(
   ref,
 ) {
   const {
-    maxWidth,
-    width,
-    minWidth,
-    w,
-    minW,
-    maxW,
     className,
     centerContent,
     ...rest
-  } = props
-
-  const theme = useTheme()
+  } = omitThemingProps(props)
 
-  const widthProps = transform(theme, {
-    maxW,
-    maxWidth,
-    width,
+  const {
     w,
-    minWidth,
-    minW,
-  })
-
-  const styles: SystemStyleObject = {
-    w: "100%",
-    mx: "auto",
-    maxW: "60ch",
-    px: "1rem",
-    ...(centerContent && {
-      display: "flex",
-      flexDirection: "column",
-      alignItems: "center",
-    }),
-    ...widthProps,
-  }
+    mx,
+    maxW,
+    px,
+    ...styles
+  } = useStyleConfig("Container", props)
 
   return (
     <chakra.div
       ref={ref}
       className={cx("chakra-container", className)}
       {...rest}
-      __css={styles}
+      __css={{
+        ...styles,
+        w: w || "100%",
+        mx: mx || "auto",
+        maxW: maxW || "60ch",
+        px: px || "1rem",
```

Comment:
Could you move the default styles to the `theme` package as you did in your last PR #2478?

Reply:
' was wondering that, too (and then got distracted instead of writing the comment to ask) - will do asap!

Reply:
Done 😄